src/model.jl

```diff
@@ -54,19 +54,47 @@ function Model(ps::Vector{Int}, qs::Vector{Int}, D::Type{<:Distribution}, scalin
     return Model{D, Float64}(ω, A, B, scaling)
 end
 
+"""
+    Model
+
+The constructor of a score-driven model, the model receives the lag structure, the 
+distribution and the scaling. You can define the lag structure in two different 
+ways, either by passing integers to define all lags. Once you build the model all of
```

Comment:
Only one way was mentioned: should probably say something like "either by passing integers `p` and `p` to add all lags from 1 to `p` and `q` or by passing vectors of integers `ps` and `qs` containing the desired lags."